lib/controller.js

```diff
@@ -941,6 +943,8 @@ controller.updateTesselWithVersion = function(opts, tessel, currentVersion, buil
 controller.tesselEnvVersions = opts => {
   opts.authorized = true;
   return controller.standardTesselCommand(opts, tessel => {
+    var pkg = path.join(__dirname, '..', 'package.json');
+    var cliVersion = fs.readJsonSync(pkg).version;
```

Comment:
The tests shouldn't be calling real `require`, you an stub it just for the test and the sandbox will restore it